zellij-utils/src/cli.rs

```diff
@@ -178,7 +178,11 @@ pub enum CliAction {
     /// [right|left|up|down]
     MovePane { direction: Direction },
     /// Dumps the pane scrollback to a file
-    DumpScreen { path: PathBuf },
+    DumpScreen {
+        path: PathBuf,
+        #[clap(short, long, value_parser)]
+        full: Option<bool>,
```

Comment:
How about using the `bool` type and `default_value` instead of using the `Option`? Like below:

```rust
#[clap(short, long, value_parser, default_value("false"), takes_value(false)]
full: bool
```